app/lib/Plugins/SearchEngine/Elastic8ConfigurationSettings.php

```diff
@@ -60,64 +60,68 @@ class Elastic8ConfigurationSettings extends ASearchConfigurationSettings {
 		}
 
-		if ( defined( '__CA_ELASTICSEARCH_INDEX_NAME__' ) && ( strlen( __CA_ELASTICSEARCH_INDEX_NAME__ ) > 0 ) ) {
+		if (defined('__CA_ELASTICSEARCH_INDEX_NAME__') && (strlen(__CA_ELASTICSEARCH_INDEX_NAME__) > 0)) {
 			$this->elasticsearch_index_name = __CA_ELASTICSEARCH_INDEX_NAME__;
 		} else {
-			$this->elasticsearch_index_name = $this->search_config->get( 'search_elasticsearch_index_name' );
+			$this->elasticsearch_index_name = $this->search_config->get('search_elasticsearch_index_name');
 		}
 
 		parent::__construct();
 	}
 
 	# ------------------------------------------------
-	public function getEngineName() {
+	public function getEngineName(): string {
 		return "Elastic8";
 	}
 
 	# ------------------------------------------------
 	private function _initMessages() {
 		# ------------------------------------------------
-		$this->setting_names[ __CA_ELASTICSEARCH_SETTING_RUNNING__ ]
-			= _t( "ElasticSearch up and running" );
-		$this->setting_names[ __CA_ELASTICSEARCH_SETTING_INDEX_EXISTS__ ]
-			= _t( "ElasticSearch index exists" );
+		$this->setting_names[__CA_ELASTICSEARCH_SETTING_RUNNING__]
+			= _t("ElasticSearch up and running");
+		$this->setting_names[__CA_ELASTICSEARCH_SETTING_INDEX_EXISTS__]
+			= _t("ElasticSearch index exists");
 		# ------------------------------------------------
-		$this->setting_descriptions[ __CA_ELASTICSEARCH_SETTING_RUNNING__ ]
-			= _t( "The ElasticSearch service must be running.", $this->webserver_user );
-		$this->setting_descriptions[ __CA_ELASTICSEARCH_SETTING_INDEX_EXISTS__ ]
-			= _t( "CollectiveAccess uses only a single index in an ElasticSearch setup. The name of that index can be set in the CollectiveAccess configuration." );
+		$this->setting_descriptions[__CA_ELASTICSEARCH_SETTING_RUNNING__]
+			= _t("The ElasticSearch service must be running.", $this->webserver_user);
+		$this->setting_descriptions[__CA_ELASTICSEARCH_SETTING_INDEX_EXISTS__]
+			= _t("CollectiveAccess uses only a single index in an ElasticSearch setup. The name of that index can be set in the CollectiveAccess configuration.");
 		# ------------------------------------------------
-		$this->setting_hints[ __CA_ELASTICSEARCH_SETTING_RUNNING__ ]
-			= _t( "Install and start the ElasticSearch service. If it is already running, check your CollectiveAccess configuration (the ElasticSearch URL and index name in particular).",
-			$this->webserver_user );
-		$this->setting_hints[ __CA_ELASTICSEARCH_SETTING_INDEX_EXISTS__ ]
-			= _t( "If the service is running and can be accessed by CollectiveAccess but the index is missing, let CollectiveAccess generate a fresh index and create the related indexing mappings. There is a tool in support/utils." );
+		$this->setting_hints[__CA_ELASTICSEARCH_SETTING_RUNNING__]
+			= _t("Install and start the ElasticSearch service. If it is already running, check your CollectiveAccess configuration (the ElasticSearch URL and index name in particular).",
+			$this->webserver_user);
```

Comment:
`$this->webserver_user` is undefined. Should this be removed? do we know what this is expected to be?

Reply:
"nuke it"